### PR TITLE
Search results

### PR DESCRIPTION
**Pull request by sfowles**
Style tweaks and light refactoring for clarity/grammar

### CODE REVIEW

File: src/Components/Scenes/SearchResults/SearchFilterChip/SearchFilterChip.js [hunk 1]
```diff
@@ -19,21 +18,22 @@ const SearchFilterChip = props => {
...
   filterQueries.forEach((filteredChip, id) => {
     const filteredBy = Object.keys(filteredChip)[0];
-    let shortNamesforContain = filteredChip[filteredBy][1];
-    let contain;
-    if (shortNamesforContain === 'in') {
-      contain = 'contain';
-    } else if (shortNamesforContain === 'eq') {
-      contain = 'equal';
+    const filteredByCap = filteredBy.charAt(0).toUpperCase() + filteredBy.slice(1);
+    let shortNamesforType = filteredChip[filteredBy][1];
```
Comment (Jake-Lindvall) [inline src/Components/Scenes/SearchResults/SearchFilterChip/SearchFilterChip.js]: Could make this a `const` for clarity, but that's also probably the most annoying nitpick I've ever written on github

**Review by Jake-Lindvall — DISMISSED** [review]
Tiny nitpick you can totally ignore, otherwise I dig!

I guess after Abby commented, to springboard off of that, there are a few small functionality changes as well that I plan to address in my search flow ticket. Closing the modal currently searches, but we don't want that (should be an easy fix). If we want, I can look into adding the no-empty-search stuff as well!

**Review by ghost — DISMISSED** [review]
Overall, LGTM!

However, there doesn't seem to be any validation if you leave the search empty. 

(mocks show red line indicating that something must be entered https://www.figma.com/file/JRTYL35e6a27C3PaP3h4xp/Kirby-UI-v2.0?node-id=1562%3A0)

Currently, it executes the search with every asset as the result. Not sure if this is a separate bug that needs to be filed or if it can be addressed here

**Comment by sfowles** [comment]
> Overall, LGTM!
> 
> However, there doesn't seem to be any validation if you leave the search empty.
> 
> (mocks show red line indicating that something must be entered https://www.figma.com/file/JRTYL35e6a27C3PaP3h4xp/Kirby-UI-v2.0?node-id=1562%3A0)
> 
> Currently, it executes the search with every asset as the result. Not sure if this is a separate bug that needs to be filed or if it can be addressed here

Good point! That's definitely not desired functionality. I actually wrote that up as a part of this ticket: https://edma.atlassian.net/secure/RapidBoard.jspa?rapidView=1&view=planning&selectedIssue=KIR-355

I'm hoping to focus on style for this particular PR, but I do think you're spot on.

There are actually a few additional issues that make this fairly janky ATM. For instance, any time a user pulls up the search modal, it loads the results component in the background; even before the user submits a search, which is also not quite what we're after.

File: src/Components/Scenes/SearchResults/SearchFilterChip/SearchFilterChip.js (hunk 1 above)
Comment (sfowles) [reply src/Components/Scenes/SearchResults/SearchFilterChip/SearchFilterChip.js]: Fixed!

**Comment by sfowles** [comment]
> Tiny nitpick you can totally ignore, otherwise I dig!
> 
> I guess after Abby commented, to springboard off of that, there are a few small functionality changes as well that I plan to address in my search flow ticket. Closing the modal currently searches, but we don't want that (should be an easy fix). If we want, I can look into adding the no-empty-search stuff as well!

Yeah really good point. I actually just quoted that exact issue in my reply to Abby lol. Also, most existing functional issues I documented in this new ticket: https://edma.atlassian.net/secure/RapidBoard.jspa?rapidView=1&view=planning&selectedIssue=KIR-355

**Comment by Jake-Lindvall** [comment]
> > Overall, LGTM!
> > However, there doesn't seem to be any validation if you leave the search empty.
> > (mocks show red line indicating that something must be entered https://www.figma.com/file/JRTYL35e6a27C3PaP3h4xp/Kirby-UI-v2.0?node-id=1562%3A0)
> > Currently, it executes the search with every asset as the result. Not sure if this is a separate bug that needs to be filed or if it can be addressed here
> 
> Good point! That's definitely not desired functionality. I actually wrote that up as a part of this ticket: https://edma.atlassian.net/secure/RapidBoard.jspa?rapidView=1&view=planning&selectedIssue=KIR-355
> 
> I'm hoping to focus on style for this particular PR, but I do think you're spot on.
> 
> There are actually a few additional issues that make this fairly janky ATM. For instance, any time a user pulls up the search modal, it loads the results component in the background; even before the user submits a search, which is also not quite what we're after.

True true. Some of the things re: routing slightly changed in the last week or two, so that led to that as well. I definitely have been thinking about this all and know how to fix in-code for the most part ❤️

**Comment by ghost** [comment]
> > Overall, LGTM!
> > However, there doesn't seem to be any validation if you leave the search empty.
> > (mocks show red line indicating that something must be entered https://www.figma.com/file/JRTYL35e6a27C3PaP3h4xp/Kirby-UI-v2.0?node-id=1562%3A0)
> > Currently, it executes the search with every asset as the result. Not sure if this is a separate bug that needs to be filed or if it can be addressed here
> 
> Good point! That's definitely not desired functionality. I actually wrote that up as a part of this ticket: https://edma.atlassian.net/secure/RapidBoard.jspa?rapidView=1&view=planning&selectedIssue=KIR-355
> 
> I'm hoping to focus on style for this particular PR, but I do think you're spot on.
> 
> There are actually a few additional issues that make this fairly janky ATM. For instance, any time a user pulls up the search modal, it loads the results component in the background; even before the user submits a search, which is also not quite what we're after.

Sounds like a plan! Thank you for making that ticket!

**Review by ghost — APPROVED** [review]
LGTM

**Review by Jake-Lindvall — APPROVED** [review]

**Pull request merged**